sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/compatibility/SpringCloudAzureSpringBootVersionVerifier.java

```diff
@@ -66,52 +70,68 @@ private boolean springBootVersionMatches() {
         return false;
     }
 
-    private Boolean bootVersionFromManifest(String s) {
+    private Optional<Boolean> bootVersionFromManifest(String s) {
```

Comment:
No need to change to Optional. You can just use `Boolean.True.equals(versionFromManifest)`